conf/addons/hpa.yaml

```diff
@@ -1,3 +1,4 @@
+{{ $max_gpus := conv.ToInt (getenv "GPU_NODE_MAX_SIZE" | default 1) }}
```

Comment:
Nice templating on this file! How do these definitions get processed again? Is it with gomplate?

Reply:
Yep, we use `gomplate` in the make command:

```
kubectl/implement/autoscaling:
	gomplate -f addons/hpa.yaml | kubectl apply -f -
```